### ISSUE TITLE
Version on About Paladin window do not match current version

### ISSUE THREAD

**Issue by OscarL** [adamfowleruk/Paladin#391]
`pkgman search -i -D paladin` shows version 2.9-1, the about window reads: Version 2.6 d0.

32/64 bits, same thing.

BTW, the "File->About Paladin" menu item should have an ellipsis (...) at the end :-)

**Comment by humdingerb** [comment]
> BTW, the "File->About Paladin" menu item should have an ellipsis (...) at the end :-)

I always find thinking about the use of ellipsis long enough and you start questioning whatever decision you've come to before... :)

Here, I'd say no ellipsis, because the result of the menu item does what the label says: the about window is shown, with no further input from the user needed. If there weren't the name of the app included, it probably should be "About...". But that's more a case of ellipsis being used for ommitting something.

I might be very wrong though, and you'll see a wild mix of ellipsis/no ellipsis when you check your installed apps...

**Comment by pulkomandy** [comment]
Yes, I agree with the "ellipsis means extra input will be needed to complete the operation" (for example "save as..." will ask for a filename). This is what we put in the UI guidelines:

> An ellipsis is a series of 3 dots (…) used to tell the user that a control, often a menu item or button, will require further input.

https://www.haiku-os.org/docs/HIG/index.xml

**Comment by OscarL** [comment]
One could argue (tongue-in-cheekly) that the resulting **modal** "about window" does requires further input (as in: click for dismissal) :-P /me ducks

**Comment by humdingerb** [comment]
I think much of the confusion comes, because at one time (was it even in the HIG, I don't remember...) it was paractised that whenever somethings opens a window, you add ellipsis.

There are a few places where even the Haiku apps don't quite follow the HIG. I may clean that up eventually...

**Comment by OscarL** [comment]
> it was paractised that whenever somethings opens a window, you add ellipsis.

Indeed, that was how I remembered/expected it. I guess I didn't got the updated HIG memo. Oh well, no biggie.

Back to see why the versions shown dot not match. :-)
